From 8de1433ad8d97750ceeb2bf2604c8e01136d8155 Mon Sep 17 00:00:00 2001
From: Zhu Honglei <zhuhonglei1714@phytium.com.cn>
Date: Wed, 20 Mar 2024 14:53:55 +0800
Subject: [PATCH 07/73] BACKPORT: PHYTIUM: edac: Phytium: Add edac driver to
 receive RAS error

Support for error detection and correction on the Phytium
Pe220x family of SOCs.

Signed-off-by: Zhu Honglei <zhuhonglei1714@phytium.com.cn>
Signed-off-by: Lan Hengyu <lanhengyu1395@phytium.com.cn>
Signed-off-by: Li Mingzhe <limingzhe1839@phytium.com.cn>
Signed-off-by: Wang Yinfeng <wangyinfeng@phytium.com.cn>
Change-Id: Iaa02a12427e8a032e12dcf148da9b6deb574a2f6

Link: https://gitee.com/phytium_embedded/phytium-linux-kernel/commit/fcb536a33d3ad3a005af1113aef503352fba964a
[Kexy: Resolved minor conflict in MAINTAINERS, dropped changes in
 drivers/edac/Kconfig, drivers/edac/Makefile,
 and drivers/edac/phytium_edac.c]
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 MAINTAINERS | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 09a0484b7..ca3ba404c 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -2894,12 +2894,6 @@ S:	Maintained
 W:	http://www.digriz.org.uk/ts78xx/kernel
 F:	arch/arm/mach-orion5x/ts78xx-*
 
-ARM/PHYTIUM SOC SUPPORT
-M:	Chen Baozi <chenbaozi@phytium.com.cn>
-S:	Maintained
-W:	https://www.phytium.com.cn
-F:	drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
-
 ARM/QUALCOMM CHROMEBOOK SUPPORT
 R:	cros-qcom-dts-watchers@chromium.org
 F:	arch/arm64/boot/dts/qcom/sc7180*
@@ -19220,6 +19214,11 @@ F:	include/sound/pxa2xx-lib.h
 F:	sound/arm/pxa*
 F:	sound/soc/pxa/
 
+ARM/PHYTIUM SOC SUPPORT
+M:	Wang Yinfeng <wangyinfeng@phytium.com.cn>
+S:	Maintained
+W:	https://www.phytium.com.cn
+
 QAT DRIVER
 M:	Giovanni Cabiddu <giovanni.cabiddu@intel.com>
 L:	qat-linux@intel.com
-- 
2.49.0

